kubert/src/admin.rs

```diff
@@ -238,3 +250,22 @@ fn handle_ready(Readiness(ready): &Readiness, req: Request<Body>) -> Response<Bo
             .unwrap(),
     }
 }
+
+fn handle_metrics(
+    metrics: &PrometheusHandle,
+    process: &Collector,
+    req: Request<Body>,
+) -> Response<Body> {
+    process.collect();
```

Comment:
nit, take it or leave it: we could probably move this call inside of the match arm for `GET` requests, so that we don't collect these metrics when we get an invalid request method?

not a big deal though.